Create SECURITY.md (#59)
geniusyield/dex-contracts-api: SECURITY.md

# Security Policy

## Reporting a Vulnerability

### Smart contracts - Bug bounty program

> As part of our commitment to the security of our smart contracts (https://github.com/geniusyield/dex-contracts-api/tree/main/geniusyield-onchain), we are inviting ethical hackers and security researchers to help us identify vulnerabilities in our smart contracts. 

Reviewer: ```suggestion
As part of our commitment to the security of [our smart contracts](https://github.com/geniusyield/dex-contracts-api/tree/main/geniusyield-onchain), we are inviting ethical hackers and security researchers to help us identify vulnerabilities in our smart contracts. 
```

By participating in our program, you will have the opportunity to earn rewards for responsibly disclosing and reporting security issues to us.

> Please find the terms and conditions, and how to submit reports here : https://immunefi.com/bug-bounty/geniusyield/

Reviewer: ```suggestion
Please find the terms and conditions, and how to submit reports on [immunefi](https://immunefi.com/bug-bounty/geniusyield/).
```

### Others 

> Vulnerability outside the smart contracts scope, please report with the built-in Github functionality in the *Security* tab:

Reviewer: ```suggestion
For any different identified vulnerabilities, outside the scope of the smart contracts; please report these using the built-in GitHub functionality under the *Security* tab:
```
